ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -533,14 +627,22 @@ public DevOptionHandler(string name, Action onSelect)
 
             public string Name { get; }
 
+#if WINDOWS_UWP
             public IAsyncInfo AsyncInfo { get; set; }
```

Comment:
>IAsyncInfo [](start = 19, length = 10)

You don't have to keep IAsyncInfo here, you can consolidate these both into a single "hide" action, and pass in asyncInfo.Cancel as the property value for UWP directly.